assets/base.css

```diff
@@ -1708,7 +1708,7 @@ input[type='checkbox'] {
   right: 0;
   bottom: 0;
   background: rgb(var(--color-background));
-  z-index: 1;
+  z-index: 4;
```

Comment:
this class is also used on the search modal from the header but from what I'm seeing it doesn't have any negative impact 🤔